index.html

```diff
@@ -20,6 +20,8 @@
 
 <body class="body">
 
+
+
```

Comment:
- [ ] OPTIONAL
Kindly ensure not to add leave empty lines in your code which are not serving any purpose, so as to keep your code cleaner